src/openrct2/localisation/string_ids.h

```diff
@@ -3647,6 +3647,16 @@ enum {
 	STR_BOOSTER = 5995,
 	STR_RIDE_CONSTRUCTION_BOOSTER_SPEED = 5996,
 
+	STR_VIEW_CLIPPING = 5997,
```

Comment:
New strings were added since you branched. You need to rebase it and up the numbers. If you need any help rebasing, feel free to ask in chat.